client/post-editor/editor-revisions/dialog.jsx

```diff
@@ -92,14 +91,14 @@ class PostRevisionsDialog extends PureComponent {
 	};
 
 	render() {
-		const { isVisible, onClose } = this.props;
+		const { isVisible, closeDialog } = this.props;
```

Comment:
We should pull out the [default prop](https://github.com/Automattic/wp-calypso/pull/19740/files#diff-b103c71eff7d16ec64b5f634effb6a5eR42) for this if we're not using it.

Looks like `Dialog` [handles](https://github.com/Automattic/wp-calypso/blob/e4c1e745befa1713e666e3526d03d58a8e2a331f/client/components/dialog/index.jsx#L51-L53) checking before calling, so we can probably just take it out.

Reply:
Nice catch, thanks! Fixed up :)